applications/nrf5340_audio/README.rst

```diff
@@ -352,6 +352,11 @@ The first part of these names is based on :kconfig:option:`CONFIG_BT_DEVICE_NAME
 
 .. _nrf53_audio_app_building:
 
+Note
+----
+Due to restrictions in the DFU system, when performing an audio DFU, there will be prints: "Failed to open flash area...".
```

Comment:
I suggest rewriting to something like this:

When performing DFU for the nRF5340 Audio application, there will be one error print related to opening flash area ID 1.
This is due to restrictions in the DFU system, and the error print is expected. The DFU process should still complete successfully.

Reply:
The error print can happen multiple times if the DFU tool used tries to do certain actions several times, so maybe something like  "one or more error prints may be logged" instead of "there will be one error print"

Reply:
@Frodevan what about now?